extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -130,6 +134,10 @@ export class LiveListManager {
       url = addParamToUrl(url, 'amp_latest_update_time',
           String(this.latestUpdateTime_));
     }
+    if (this.lastCheckTime_ > 0) {
+      url = addParamToUrl(url, 'amp_last_check_time',
```

Comment:
can we add a comment here that explains that this is necessary for cache busting etc. I'll merge right after.

I really appreciate the fix submission!